### PR TITLE
net/keepalived: update version and add hotplug.d notify handling

### PR DESCRIPTION
**Pull request by feckert**
Maintainer: me / @scrpi 
Compile tested: x86_64 and lantiq_xrx200 , lede-17.01
Run tested: x86_64 and lantiq_xrx200, lede-17.01, 

Description:

- Update keepalived to the newest upstream version 1.3.9.
- Add auto hotplug.d handling on for notificaions
- Add keepalived.user hotplug script
- Comment alt_config_file option selection in default config

### CODE REVIEW

File: net/keepalived/files/keepalived.init [hunk 1]
```diff
@@ -242,8 +254,11 @@ vrrp_sync_group() {
 	group="$valid_group"
 	print_list_indent group
 
-	print_elems_indent $1 $INDENT_1 notify_backup notify_master notify_fault \
-		notify no_val_smtp_alert no_val_global_tracking
+	print_elems_indent $1 $INDENT_1 no_val_smtp_alert no_val_global_tracking
+
+	print_notify "GRPOUP" "$name" notify_backup notify_master \
```
Comment (scrpi) [inline net/keepalived/files/keepalived.init]: looks like a typo here - `GRPOUP`

Comment (feckert) [reply net/keepalived/files/keepalived.init]: @scrpi fixed thanks

**Comment by champtar** [comment]
closing/openning to trigger travis

**Comment by feckert** [comment]
@scrpi I have some additional change for this package! Should i force this PR? to get reviewed or should i open an other PR if this get hopefully get merged.

**Comment by scrpi** [comment]
@feckert I think we should open a second future PR with the additional changes - have approved this one now though someone with write access to the repo will need to help merge. Thanks

**Comment by feckert** [comment]
@scrpi thanks for reviewing my changes. I will to a follow PR in the next days with my additional changes.
@champtar can you merge this?

**Comment by champtar** [comment]
Hi @feckert 
Travis refuse to build saying "abuse detected"
https://travis-ci.org/openwrt/packages/requests
I'm merging, but please have a look at the buildbot results

**Comment by champtar** [comment]
Just sent an email to Travis support

**Comment by feckert** [comment]
@champtar I cannot find any issue on this commit travis complains!

**Comment by champtar** [comment]
Hi @feckert,
Here is the response from Travis:

> Looking at the organization that the PR was opened from (i.e. TDT-GmbH), it seems that nobody with push access to that repo has ever logged into Travis CI and this is why we are currently blocking your requests. Sorry for the inconvenience.
> 
> To confirm, do you think you could try to log into Travis CI at least once at https://travis-ci.org/ and then let us know? Once you do that, we can remove the abusive flag from the org, and then you can close and reopen the PR for it to build successfully.
> 

Can you try to login to Travis and ping me?

**Comment by feckert** [comment]
@champtar i have logged in and granted TDT-GmbH?

**Pull request merged**